src/log.ts

```diff
@@ -53,9 +88,26 @@ function getStepStatusText(status: IntegrationStepResultStatus) {
   }
 }
 
-export function displaySynchronizationResults(job: SynchronizationJob) {
-  info('\nSynchronization results:\n');
-  info(`Synchronization job status: ${chalk.cyan(job.status)}`);
-  info(`Entities uploaded: ${chalk.cyan(job.numEntitiesUploaded)}`);
-  info(`Relationships uploaded: ${chalk.cyan(job.numRelationshipsUploaded)}`);
+function logUndeclaredAndExtraneousTypes(
```

Comment:
I'm not sure about the use of `And`, because I keep thinking (when reading the tests as well) that this is referring to two kinds of types, some that are undeclared and some that are extraneous, though I think it is referring only to a single kind of type: those which appeared in the collection but were not declared. Additionally, the data is tracked as `undeclaredTypes`, not `undeclaredAndExtraneousTypes`.

What do you think about just `logUndeclaredTypes` (and removing "extraneous" elsewhere as well)?

Reply:
Updated. I originally had extraneous types being logged later decided to stop doing so because for some integration developers that have a different provider setup, it could cause end up causing unnecessary noise.